src/push/request.ts

```diff
@@ -80,14 +80,11 @@ export async function handleError(
     throw formatNotFoundError(url, await body.text());
   } else if (!ok(statusCode)) {
     let parsed: APIError;
+    const resp = await body.text();
     try {
```

Comment:
```suggestion

    const resp = await body.text();

```

Reply:
We should guard against `body.text()` rejections

Reply:
Good catch. I will move the error message as generic as well.